sky/skylet/constants.py

```diff
@@ -57,6 +57,13 @@
 SKY_UV_PIP_CMD = f'VIRTUAL_ENV={SKY_REMOTE_PYTHON_ENV} {SKY_UV_CMD} pip'
 # Deleting the SKY_REMOTE_PYTHON_ENV_NAME from the PATH to deactivate the
 # environment. `deactivate` command does not work when conda is used.
+
+# Define SKY_GET_PIP_CMD to try uv first, fall back to regular pip if uv is not
+# available. This is only for backwards compatibility with pre-nimbus SkyPilot
+# versions.
+# TODO(romilb): Change to using SKY_UV_PIP_CMD after v0.10.0.
+SET_SKY_PIP = f'SKY_PIP() {{ which uv >/dev/null 2>&1 && {SKY_UV_PIP_CMD} "$@" || {SKY_PIP_CMD} "$@"; }};'  # pylint: disable=line-too-long
```

Comment:
Did we add uv in path? Should we use `which {SKY_UV_CMD}` instead?

Reply:
With uv changes reverted in #4401, this is no longer applicable